actioncable/lib/action_cable/engine.rb

```diff
@@ -31,6 +32,17 @@ class Railtie < Rails::Engine # :nodoc:
           self.cable = Rails.application.config_for(config_path).with_indifferent_access
         end
 
+        if 'ApplicationCable::Connection'.safe_constantize
+          self.connection_class = ApplicationCable::Connection
+        else
+          raise RuntimeError, <<-eow.strip_heredoc
+            Unable to find an `ApplicationCable::Connection` for Action Cable to use.
+            Check the README for instructions on where and how to set it up.
```

Comment:
Quirky error message. Should be removed when we change `rails:update` to generate the `connection.rb` file.